KarhooUISDK/Screens/CheckoutScreen/Components/Loyalty/LoyaltyView.swift

```diff
@@ -89,9 +108,16 @@ struct LoyaltyView: View {
                     .fixedSize(horizontal: false, vertical: true)
             }
             .layoutPriority(Double(UILayoutPriority.defaultHigh.rawValue))
+            .accessibilityElement()
+            .accessibilityValue(UITexts.Loyalty.burnTitle + "." + (viewModel.isBurnModeOn ? burnOnSubtitle : viewModel.burnOffSubtitle))
             Toggle("", isOn: $viewModel.isBurnModeOn.animation())
                 .toggleStyle(SwitchToggleStyle(tint: Color(KarhooUI.colors.secondary)))
                 .disabled(viewModel.burnSectionDisabled)
+                .accessibilityValue(
```

Comment:
This doesn't work for me when I tap on the switch. It says "Switch button. Double tap to toggle setting" no matter the mode.